layers/meta-resin-raspberrypi/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -3,7 +3,14 @@ UBOOT_KCONFIG_SUPPORT = "1"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+PREFERRED_VERSION_python = "2018.07"
```

Comment:
Why do you need this? Do we have another set preferred version?

Reply:
why does that say python lol. that should say PREFERRED_VERSION_u-boot!

lol